storage/tests/unit/test_bucket.py

```diff
@@ -223,30 +223,30 @@ def test_from_api_repr_w_empty_resource(self):
         config = klass.from_api_repr(resource, bucket)
 
         self.assertIs(config.bucket, bucket)
-        self.assertFalse(config.bucket_policy_only_enabled)
-        self.assertIsNone(config.bucket_policy_only_locked_time)
+        self.assertFalse(config.uniform_bucket_level_access_enabled)
+        self.assertIsNone(config.uniform_bucket_level_access_locked_time)
 
-    def test_from_api_repr_w_empty_bpo(self):
+    def test_from_api_repr_w_empty_ubla(self):
         klass = self._get_target_class()
         bucket = self._make_bucket()
-        resource = {"bucketPolicyOnly": {}}
+        resource = {"uniformBucketLevelAccess": {}}
 
         config = klass.from_api_repr(resource, bucket)
 
         self.assertIs(config.bucket, bucket)
-        self.assertFalse(config.bucket_policy_only_enabled)
-        self.assertIsNone(config.bucket_policy_only_locked_time)
+        self.assertFalse(config.uniform_bucket_level_access_enabled)
+        self.assertIsNone(config.uniform_bucket_level_access_locked_time)
```

Comment:
We need to retain the assertions about the deprecated properties here.

Also, do we need a separate testcase for a resource which contains only an empty `bucketPolicyOnly` mapping?  I.e., can we guarantee that the back-end won't return such a resource?